backend/Application/Api/Rest/BalanceStatisticsController.cs

```diff
@@ -31,21 +31,33 @@ public async Task<ActionResult> GetLatest(string field)
         var scalarQuery = field.ToLowerInvariant() switch
         {
             "totalamount" => query.Select(x => (ulong?)x.BalanceStatistics.TotalAmount),
-            "totalamountreleased" => query.Select(x => x.BalanceStatistics.TotalAmountReleased),
+            "totalamountcirculating" => query.Select(x => x.BalanceStatistics.TotalAmountReleased),
+            "totalamoununlocked" => query.Select(x => x.BalanceStatistics.TotalAmountUnlocked),
             "totalamountnotreleased" => query.Select(x => x.BalanceStatistics.TotalAmount - x.BalanceStatistics.TotalAmountReleased),
             "totalamountstaked" => query.Select(x => (ulong?)x.BalanceStatistics.TotalAmountStaked),
             "totalamountencrypted" => query.Select(x => (ulong?)x.BalanceStatistics.TotalAmountEncrypted),
             "totalamountlockedinreleaseschedules" => query.Select(x => (ulong?)x.BalanceStatistics.TotalAmountLockedInReleaseSchedules),
             _ => throw new ArgumentOutOfRangeException(nameof(field), field, "Supported values for field are: ")
         };
-        var value = await scalarQuery.FirstAsync();    
+
+        var value = await scalarQuery.FirstAsync();
+        var retValue = "";
+        if (!String.IsNullOrWhiteSpace(unit) && value.HasValue)
+        {
+            retValue = unit.ToLowerInvariant() switch
+            {
+                "ccd" => retValue = ((decimal)value / 10_00_000).ToString(CultureInfo.InvariantCulture),
+                "microccd" => ((decimal)value).ToString(CultureInfo.InvariantCulture),
+                _ => ((decimal)value).ToString(CultureInfo.InvariantCulture)
+            };
+        }
```

Comment:
We are returning `""`(Empty String) because thats what the previous code was returning when there was no value present. 
We can also here assume that if String.IsNullOrWhiteSpace(unit) then `unit` = ccd.
unit can be null or whitespace because its being read from Query Strings and can be changed / removed by the caller.